setup.cfg

```diff
@@ -54,6 +54,7 @@ include_package_data = True
 packages = find:
 python_requires = >=3.7
 install_requires =
+    typing_extensions
```

Comment:
it would be great if we could require this only for a pthon version lower of a certain version. 

Since at the moment we need ParamSpec I guess we could make it install only for < 3.10

Reply:
Yes, we want to keep Falcon free from third party dependencies.
It is OK to add dependencies for soon EOL 3.8 or 3.9.